godot-codegen/src/special_cases/codegen_special_cases.rs

```diff
@@ -147,6 +147,7 @@ const SELECTED_CLASSES: &[&str] = &[
     "InputEventAction",
     "Label",
     "MainLoop",
+    "Material",
```

Comment:
Can we avoid this somehow? Every class added here increases CI time.

Reply:
Clippy checks with default features but `itest` disables them, so depending on which of the two you run there are different complaints. In one case `IPrimitiveMesh::surface_set_material` and `IPrimitiveMesh::surface_get_material` are there, in the other they are not.

Maybe making `codegen-full` a default feature of `itest` and disabling default features in the CI build could work.

Reply:
I solved this as described.